torch/nn/quantized/modules/activation.py

```diff
@@ -149,3 +149,24 @@ def _get_name(self):
     def from_float(cls, mod):
         scale, zero_point = mod.activation_post_process.calculate_qparams()
         return cls(float(scale), int(zero_point), mod.negative_slope, mod.inplace)
+
+class Sigmoid(torch.nn.Sigmoid):
+    r"""This is the quantized equivalent of :class:`~torch.nn.LeakyReLU`.
+
+    Args:
+        scale: quantization scale of the output tensor
+        zero_point: quantization zero point of the output tensor
+    """
+
+    def __init__(self, output_scale: float, output_zero_point: int):
```

Comment:
Can we have default args? It is safe to assume those for this function

Reply:
not sure, it's not likely that user is going to construct quantized model from quantized modules from scratch anyways, what is the benefit of having default args?